src/openpodcast.js

```diff
@@ -1,25 +1,39 @@
+const runtime = window.browser || chrome
+
 //insert html at the beginning of the website
 function showCredentialsOnWebsite(creds) {
-  let message = "No Spotify credentials found, are you logged in on podcasters.spotify.com? (reload needed)"
+  let message = "No Spotify credentials found, are you logged in on podcasters.spotify.com? (maybe reload this page)"
   if (creds.sp_key && creds.sp_dc) {
-    message = `Spotify Credentials: key: ${creds['sp_key']}, dc: ${creds['sp_dc']}`
+    message = `Spotify keys: key: ${creds['sp_key']}, dc: ${creds['sp_dc']} END`
```

Comment:
`keys: key` can be confusing. How about `Spotify cookies: key`?

Reply:
we have to clean this later anyways. cookie is also not well understood by people. for now it should be fine, I also didn't want to trigger the reviewer by reading "cookie" information.